examples/evm/its-canonical-token/CanonicalToken.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { ERC20 } from '@openzeppelin/contracts/token/ERC20/ERC20.sol';
+
+import { Minter } from '@axelar-network/interchain-token-service/contracts/utils/Minter.sol';
```

Comment:
This totally works, but one of our goals is around reducing mental overhead. Using OpenZeppelin instead of our own custom access control system means less code to review/teach. 

I can do the work to migrate after this PR, but what do you think?

Reply:
yea you can simplify as you see fit.